frontend/src/lib/i18n/translations/en/candidateApp.json

```diff
@@ -20,7 +20,7 @@
   },
   "logoutModal": {
     "title": "Some of Your Data Is Still Missing",
-    "body": "There are still 10 items of basic info and 25 opinions to fill. Your data won't be shown in the Election Compass until you have filled these, but you can login later to continue.",
+    "body": "There are still {remainingInfoAmount} items of basic info and {remainingOpinionNumber} opinions to fill. Your data won't be shown in the Election Compass until you have filled these, but you can login later to continue.",
```

Comment:
In the case that one of the numbers is 0, would it be possible to leave that part out, i.e. "there are still 0 items of basic info and 3 opinions" -> "there are still 3 opinions to fill"